js/basicblocks.js

```diff
@@ -3743,6 +3743,23 @@ function initBasicProtoBlocks(palettes, blocks, beginnerMode) {
         newblock.hidden = true;
     }
 
+    var newblock = new ProtoBlock('distance');
+    newblock.palette = palettes.dict['number'];
+    blocks.protoBlockDict['plus'] = newblock;
+    newblock.fontsize = 28;
+    newblock.staticLabels.push('||');
```

Comment:
maybe the label should be distance. And we want to add labels for the args: x1, y1, x2, y2

Reply:
What changes need to be done ? Do I need to change only js/basicblocks.js ?